src/main/java/htsjdk/samtools/util/IntervalListWriter.java

```diff
@@ -41,18 +42,18 @@
     private final FormatUtil format = new FormatUtil();
 
     /** Creates a new writer, writing a header to the file.
-     * @param file a file to write to.  If exists it will be overwritten.
+     * @param path a path to write to.  If exists it will be overwritten.
      */
-    public IntervalListWriter(final File file) {
-        this(file, null);
+    public IntervalListWriter(final Path path) {
+        this(path, null);
     }
 
     /** Creates a new writer, writing a header to the file.
-     * @param file a file to write to.  If exists it will be overwritten.
+     * @param path a file to write to.  If exists it will be overwritten.
      * @param header the header to write.
      */
-    public IntervalListWriter(final File file, final SAMFileHeader header) {
-        out = IOUtil.openFileForBufferedWriting(file);
+    public IntervalListWriter(final  Path path, final SAMFileHeader header) {
+        out = IOUtil.openFileForBufferedWriting(path.toFile());
```

Comment:
This kind of defeats the purpose since it means that you can't actually use generic Paths, only ones that represent files.  I hadn't realized that we were missing path equivalents of this method though.  I've opened #1294 and https://github.com/samtools/htsjdk/pull/1296 to fix it.